distrobox-create

```diff
@@ -118,9 +124,13 @@ if [ -z "${container_clone}" ] && [ -z "${container_image}" ]; then
 	container_image="${container_image_default}"
 fi
 
+if [ "${container_name}" == "" ]; then
```

Comment:
This behavior is not defined in POSIX sh, probably it's better to check if it's empty using

`if [ -z  "${container_name}" ]; then`